slackclient/server.py

```diff
@@ -150,11 +141,7 @@ def rtm_connect(self, reconnect=False, timeout=None, use_rtm_start=True, **kwarg
             else:
                 self.reconnect_count = 0
 
-        reply = self.api_requester.make_http_request(
-            api_method=connect_method,
-            timeout=timeout,
-            post_data=kwargs
-        )
+        reply = self.api_requester.post_http_request(self.token, connect_method, kwargs)
```

Comment:
any advantage to using `post_http_request` over `do`? i think the latter is probably better for consistency (since `post_http_request` feels like it should be reserved for places where you want specifically low level control without the possibility of a retry)